<launch>


  <!-- load parameters to parameter server through yaml file -->
 

  <rosparam file="$(find path_planner)/config/planner_interface.yaml" />
  <arg name="node_start_delay" default="20.0" />  
  

  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find path_planner)/planner_rviz_config.rviz" />
  <node name="path_planner" pkg="path_planner" type="planner_interface" output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' " >
  	
   

  </node>


   



</launch>
